back/lib/finder/sortable.rb

```diff
@@ -28,8 +28,7 @@ def sort_scope(scope_name, blk_or_options)
       def default_sort(scope)
         @_default_sort, @_default_sort_order = scope.first if scope.is_a? Hash
 
-        @_default_sort_order = scope.to_s.delete_prefix!('-') ? :asc : :desc
```

Comment:
This line was effectively removing any hyphen as prefix from the scope, meaning a [later conditional](https://github.com/CitizenLabDotCo/citizenlab/blob/620be400b32ff9333cd6dae0c49112352ab55c1d/back/lib/finder/sortable.rb#L97) of the presence of such a hyphen, in `def _sort_order` was always false and thus always returned 'asc' regardless of whether the value set for `default_sort` in[ events_finder.rb](https://github.com/CitizenLabDotCo/citizenlab/blob/master/back/app/finders/events_finder.rb) contained a hyphen prefix or not.

Reply:
Nice find Simon!

Reply:
Indeed a nice find. `delete_prefix!` was removed earlier when we merged RuboCop changes. It must have slipped back in somehow later. See https://github.com/CitizenLabDotCo/citizenlab/commit/60f26470 and see the comment I added back then.

Reply:
It was Alex who finally spotted this line was the culprit. He deserves the credit.